scripts/run_experiment.py

```diff
@@ -186,5 +202,34 @@ def run_experiment(argv):
     logger.pop_prefix()
 
 
+def child_proc_shutdown(shutdown_sampler=False):
```

Comment:
If you change the Plotter API to use `close()` (or the sampler API to use `shutdown()`) then this could be called as

```python
def child_proc_shutdown(children):
    run_exp_proc = psutil.Process()
    alive = run_exp_proc.children(recursive=True)
    for c in children:
        c.shutdown()
    # etc...

# example at the callsite
child_proc_shutdown(__plotters__ + [parallel_sampler])
```